### ISSUE TITLE
enable_profiler => true shows php notices (undefined index)

### ISSUE THREAD

**Issue by sandrodz** [pug-php/pug#236]
Hello,

Recently tried enable_profiler parameter and got some undefined index errors:

```
Notice: Undefined index: timeprecision in /vendor/phug/phug/src/Phug/Util/Util/Partial/OptionTrait.php on line 226

Notice: Undefined index: lineheight in /vendor/phug/phug/src/Phug/Util/Util/Partial/OptionTrait.php on line 226
```

Thanks!

**Comment by kylekatarnls** [comment]
Hello,

I did:
```php
<?php

include __DIR__.'/vendor/autoload.php';

error_reporting(E_NOTICE);

Phug::setOptions([
    'enable_profiler' => true,
]);

Phug::display('p Hello');
```

And I got no notice.

I need the code you run and the complete stack traces. 

Thanks.

**Comment by kylekatarnls** [comment]
OK,

I see it happens with:

```php
<?php

include __DIR__.'/vendor/autoload.php';

error_reporting(E_NOTICE);

$pug = new Pug([
    'enable_profiler' => true,
]);

$pug->display('p Hello');
```

Indeed I have to map the options from Pug to Phug.

**Comment by sandrodz** [comment]
Hey @kylekatarnls ! 

Have a look, this is where I run it:
https://github.com/kubikplugins/pugpress/blob/master/src/class-pugpress.php#L167-L175

And options:
https://github.com/kubikplugins/pugpress/blob/master/src/class-pugpress.php#L77-L86

**Comment by kylekatarnls** [comment]
It should be fixed in the next version, you can try it requiring `"pug-php/pug": "dev-master"`

**Comment by kylekatarnls** [comment]
Released in 3.4.1. Thanks!

**Comment by sandrodz** [comment]
Thank you. Your awlays super efficient and fast.
